packages/react-lowcode/src/codegen/app.ts

```diff
@@ -57,7 +54,7 @@ export function generatePages(introspection: IntrospectionQuery, io: CodeRW & Co
       io.writeFile(filePath, pageSourceCode)
 
       //generate list wrapper
-      const templateResolver = new TemplateResolver(entity);
+      const templateResolver = new TemplateResolver(introspection, entity);
```

Comment:
ten introspection dajme ako druhy a zaroven nepovinny argument (su miesta kde sa to uz pouziva)
https://github.com/iteria-app/lowcode/blob/d5739cc4dca5adfdb5929e9be4c675f0ecdb1281/packages/react-lowcode/src/codegen/tests/template/template.test.ts

spusti si aj testy prosim na localhoste